### ISSUE TITLE
PicoMiteWeb Wifi no conect

### ISSUE THREAD

**Issue by Surac** [UKTailwind/PicoMiteWeb#1]
using the PicoMiteWeb 5.07.07RC8

I am able to connect my wifi by using OPTION WEB ... from console. Everything works fine.
After power down/up cycle the PicoMiteWeb board I only get a error message:

WebMite MMBasic Version 5.07.07RC8
Copyright 2011-2023 Geoff Graham
Copyright 2016-2023 Peter Mather

Connecting to WiFi...
failed to connect.

Using a FritzBox as Wlan AP

**Comment by UKTailwind** [comment]
Fritzbox are a pile of poo. Other users reports trying to connect a second time without power cycling will then work. No other make of router has this problem

**Comment by Surac** [comment]
Yes, it works if I use OPTION WIFI from the PicoMiteWeb command prompt, even after getting the error message on power up.

But... it works like a charm if i use the C++ SDK from Raspberry. Never got any connection problems in my C projects

Also... Fritz Boxes are widely used here in Germany and i cannot make all sales undone by naming them are pile of poo :)

is there a way to get more information what goes wrong while it tries to connect?

**Comment by UKTailwind** [comment]
You can check my code in PicoMite.c,  function WebConnect but it doesn't do anything other than proposed in the various examples. I haven't found any way to diagnose wifi issues as WireShark doesn't support wifi.
